tests/test_entity_ranking.py

```diff
@@ -0,0 +1,116 @@
+import unittest
+import tempfile
+import yaml
+
+import torch
+from kge.job import EntityRankingJob
+from torch import Tensor
+
+from kge import Dataset
+from kge.model import KgeModel
+
+from tests.util import create_config, get_dataset_folder
+
+
+class TestModel(KgeModel):
```

Comment:
Call this differently (it's not a test, but a dummy model).

Reply:
The easiest way would be to use a wrapper model that takes a model and perturbs it's spo score. This then takes just a few lines.